files/fr/web/api/document/execcommand/index.md

```diff
@@ -84,7 +84,7 @@ prise en charge ou si elle est désactivée.
 - `foreColor`
   - : Modifie la couleur de la police pour la sélection ou à partir du point d'insertion. La valeur passée en argument doit être une chaîne de caractères décrivant une couleur exprimée en notation hexadécimale.
 - `formatBlock`
-  - : Ajoute un élément HTML de bloc autour de la ligne contenant la sélection courante et remplace l'éventuel élément de bloc englobant s'il en existe un (pour Firefox, {{HTMLElement("blockquote")}} fera exception et englobera n'importe quel bloc). L'argument passé doit être un nom de balise. Tous les éléments de bloc peuvent en théorie être utilisés. (Internet Explorer et Edge ne prennent en charge que les balises `H1`-`H6`, `ADDRESS`, et `PRE` qui doivent être passées entre chevrons : `"<H1>"`.)
+  - : Ajoute un élément HTML de bloc autour de la ligne contenant la sélection courante et remplace l'éventuel élément de bloc englobant s'il en existe un (pour Firefox, {{HTMLElement("blockquote")}} fera exception et englobera n'importe quel bloc). L'argument passé doit être un nom de balise. Tous les éléments de bloc peuvent en théorie être utilisés. (Internet Explorer et Edge ne prennent en charge que les balises `H1` - `H6`, `ADDRESS`, et `PRE` qui doivent être passées entre chevrons : `"<H1>"`.)
```

Comment:
This one looks like a range again, I suppose I don't really mind it since they are not quite numbers but maybe:

```suggestion
  - : Ajoute un élément HTML de bloc autour de la ligne contenant la sélection courante et remplace l'éventuel élément de bloc englobant s'il en existe un (pour Firefox, {{HTMLElement("blockquote")}} fera exception et englobera n'importe quel bloc). L'argument passé doit être un nom de balise. Tous les éléments de bloc peuvent en théorie être utilisés. (Internet Explorer et Edge ne prennent en charge que les balises `H1` à `H6`, `ADDRESS`, et `PRE` qui doivent être passées entre chevrons : `"<H1>"`.)
```

to simply avoid dashes